lib/internal/perf/observe.js

```diff
@@ -97,23 +96,18 @@ function queuePending() {
   });
 }
 
+function getObserverType(type) {
+  switch (type) {
+    case 'gc': return NODE_PERFORMANCE_ENTRY_TYPE_GC;
+    case 'http2': return NODE_PERFORMANCE_ENTRY_TYPE_HTTP2;
+    case 'http': return NODE_PERFORMANCE_ENTRY_TYPE_HTTP;
+  }
```

Comment:
This is not identical to before as there's no `function` type anymore. Is that intentional?

```suggestion
    case 'function': return NODE_PERFORMANCE_ENTRY_TYPE_FUNCTION;
  }
```

Reply:
Yes, the function type here was removed. The original refactor missed removing it

Reply:
Maybe add that in the commit description? :-)

Reply:
Done